index.html

```diff
@@ -215,13 +311,14 @@ <h3 class="language">Skills</h3>
               <li>Gitlab</li>
               <li>Terminal</li>
             </ul>
+        </section>
       </section>
     </main>
   
     <footer> 
       <section class="contact">
         <p>I'm always interested in hearing about new projects,
-        so if you'd like to chat please get in</p>
+        so if you'd like to chat please get in touch</p>
         <form action="https://formspree.io/f/xjvzpjgq" method="post">
```

Comment:
- [ ] Please align the buttons and form at the same line and you should stick to the design as much as possible. Please see image below for reference
### Your design
![his](https://user-images.githubusercontent.com/81727988/199898203-7a30573b-c573-4f49-8561-2302aa0a0213.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/199898225-caa8d8ca-d1de-4ec3-a86c-d89582c53a96.png)